staging/src/k8s.io/client-go/pkg/apis/apps/types.go

```diff
@@ -60,6 +60,50 @@ const (
 	ParallelPodManagement = "Parallel"
 )
 
+// StatefulSetUpdateStrategy indicates the strategy that the StatefulSet
+// controller will use to perform updates. It includes any additional parameters
+// necessary to preform the update for the indicated strategy.
+type StatefulSetUpdateStrategy struct {
```

Comment:
No. You have to add your internal and external types. These are what everything else is generated from.